src/guides/Filament-Manager-Installation.md

```diff
@@ -0,0 +1,122 @@
+# Setup guide for remote database using the plugin FilamentManager
```

Comment:
Would you mind adding a link to the actual plugin somewhere under here?

Format:  
`## [Cost Estimation](https://github.com/OllisGit/OctoPrint-CostEstimation)`